Rakefile

```diff
@@ -164,3 +164,40 @@ task :c2m_check_version_all_dirs, [:last_checked_b4_date, :profile] => [:environ
   end
   $stdout.flush
 end
+
+desc "Fire off checksum validation on a single endpoint"
+task :cv_single_endpoint, [:last_checked_b4, :endpoint, :algorithm, :profile] => [:environment] do |_t, args|
+  unless args[:profile] == 'profile' || args[:profile].nil?
+    p "usage: rake cv_single_endpoint[last_checked_b4, endpoint, algorithm] || rake cv_single_endpoint[last_checked_b4, endpoint, algorithm, profile]"
+    exit
+  end
+  last_checked = args[:last_checked_b4].to_sym
+  endpoint = args[:endpoint].to_sym
+  algorithm = args[:algorithm].to_sym
+  if args[:profile] == 'profile'
+    puts "When done, check log/profile_cv_single_endpoint2018-02-02T21:37:23Z.txt for profiling details"
```

Comment:
I think this won't be accurate when the task is run. Maybe interpolate the date argument? Something like:

```
puts "When done, check log/profile_cv_single_endpoint#{last_checked}.txt for profiling details"
```